instance/tests/integration/integration_instance.py

```diff
@@ -78,7 +78,7 @@ def assert_appserver_firewalled(self, instance):
         instance.refresh_from_db()
         self.assertIsNotNone(instance.active_appserver)
         server_ip = instance.active_appserver.server.public_ip
-        ports_should_be_open = [22, 80]  # 443 may or may not be open, depending on instance.protocol
+        ports_should_be_open = [22, 80]  # 443 may or may not be open
```

Comment:
Per previous comment, maybe this should always be open? If you don't make that change then this comment is wrong - it's always closed.